skeleton.cpp

```diff
@@ -1024,6 +1026,7 @@ void Skeleton::UponTestDateConversions(wxCommandEvent&)
     TestDateConversions();
 }
 
+#if defined LMI_X87
```

Comment:
It didn't seem useful to neither keep a test doing nothing nor creating a (necessarily) completely different and artificial version of this test for non-x87.